includes/wc-term-functions.php

```diff
@@ -395,11 +395,21 @@ function wc_reorder_terms( $the_term, $next_id, $taxonomy, $index = 0, $terms =
 		// set order
 		$index++;
 		$index = wc_set_term_order( $term->term_id, $index, $taxonomy );
+		
+		/**
+		 * After a term has had it's order set.
+		*/
+		do_action( 'woocommerce_set_term_order', $term, $index, $taxonomy );
```

Comment:
Figured the lower down in the call hirarchy its would be placed, the more the action would run = wasted cpu cycles. Didn't place it further down in case a lot of other places would use wc_set_term_order in ways unrelated to ajax sorting. This function is called directly from WC_Ajax.